mlir/lib/Dialect/Rock/utility/AmdArchDb.cpp

```diff
@@ -118,7 +118,7 @@ GemmFeatures mlir::rock::AmdArchInfo::getDefaultFeatures(Type dataType) {
   }
   bool isMfma = bitEnumContainsAll(theseFeatures, GemmFeatures::mfma);
   if (isMfma && !hasFp8ConversionInstrs) {
-    if (dataType.isFloat8E4M3FNUZ() || dataType.isFloat8E5M2FNUZ())
+    if (isa<FloatType>(dataType) && dataType.getIntOrFloatBitWidth() == 8)
```

Comment:
Another 8-bit-floats acceptance point, but not worth making a function for the one place.